chowpronto_frontend/src/shared_components/ChowButton.js

```diff
@@ -4,16 +4,7 @@ import PropTypes from "prop-types";
 
 const ChowButton = (props) => {
   const { title, onClick, primary, secondary, tertiary, elevated } = props;
-  return (
-    <StyledButton
-      onClick={onClick}
-      primary={primary}
-      secondary={secondary}
-      tertiary={tertiary}
-    >
-      {title}
-    </StyledButton>
-  );
+  return <StyledButton {...props}>{title}</StyledButton>;
```

Comment:
Is the destructuring of onClick, primary, etc. for later?

Reply:
Yes greg, this is a bit of a weird thing with using SC but I have switched over to passing {...props} on newer branch (add-place-order-button) which I will push later today.

Reply:
No worries 😄